sdk/spring/azure-spring-boot-test-b2c/src/test/java/com/azure/test/b2c/selenium/B2CSeleniumITHelper.java

```diff
@@ -0,0 +1,158 @@
+import java.util.HashMap;
+import java.util.Map;
+import java.util.regex.Pattern;
+import org.junit.Assert;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeDriverService;
+import org.openqa.selenium.chrome.ChromeOptions;
+
+public class B2CSeleniumITHelper {
+
+    private final String emailAddress;
+    private final String password;
+    private final AppRunner app;
+    private final WebDriver driver;
+    private static final Map<String, String> DEFAULT_PROPERTIES = new HashMap<>();
+
+    static {
+        DEFAULT_PROPERTIES.put("azure.activedirectory.b2c.tenant", B2C_TENANT);
+        DEFAULT_PROPERTIES.put("azure.activedirectory.b2c.client-id",  System.getenv(B2C_CLIENT_ID));
+        DEFAULT_PROPERTIES.put("azure.activedirectory.b2c.client-secret",  System.getenv(B2C_CLIENT_SECRET));
+        DEFAULT_PROPERTIES.put("azure.activedirectory.b2c.reply-url", B2C_REPLY_URL);
+        DEFAULT_PROPERTIES
+            .put("azure.activedirectory.b2c.user-flows.sign-up-or-sign-in", B2C_SIGN_UP_OR_SIGN_IN);
+        DEFAULT_PROPERTIES.put("azure.activedirectory.b2c.user-flows.profile-edit", B2C_PROFILE_EDIT);
+
+        final String directory = "src/test/resources/driver/";
+        final String chromedriverLinux = "chromedriver_linux64";
+        final String chromedriverWin32 = "chromedriver_win32.exe";
+        final String chromedriverMac = "chromedriver_mac64";
+        String osName = System.getProperty("os.name").toLowerCase();
+        Process process = null;
+        try {
+            File dir = new File(directory);
+            if (Pattern.matches("linux.*", osName)) {
+                process = Runtime.getRuntime().exec("chmod +x " + chromedriverLinux, null, dir);
+                process.waitFor();
+                System.setProperty(ChromeDriverService.CHROME_DRIVER_EXE_PROPERTY, directory + chromedriverLinux);
+            } else if (Pattern.matches("windows.*", osName)) {
+                System.setProperty(ChromeDriverService.CHROME_DRIVER_EXE_PROPERTY, directory + chromedriverWin32);
+            } else if (Pattern.matches("mac.*", osName)) {
+                process = Runtime.getRuntime().exec("chmod +x " + chromedriverMac, null, dir);
+                process.waitFor();
+                System.setProperty(ChromeDriverService.CHROME_DRIVER_EXE_PROPERTY, directory + chromedriverMac);
+            } else {
+                throw new IllegalStateException("Unrecognized osName. osName = " + System.getProperty("os.name"));
+            }
+        } catch (InterruptedException | IOException e) {
+            throw new RuntimeException(e);
+        } finally {
+            if (process != null) {
+                process.destroy();
+            }
+        }
+    }
+
+    public B2CSeleniumITHelper(Class<?> appClass, Map<String, String> properties) throws InterruptedException {
+        emailAddress = B2C_USER_EMAIL;
```

Comment:
how about change `emailAddress` to `userEmail` and `password` to `userPassword` to make them consistent.